Scroller/EditableWrapper.js

```diff
@@ -192,14 +216,18 @@ const EditableWrapper = (props) => {
 	}, [dataSize, addRearrangedItems, removeRearrangedItems]);
 
 	// Remove an item
-	/* const removeItem = useCallback(() => {
-		const {prevToIndex} = mutableRef.current;
-		const orders = finalizeOrders();
+	const removeItem = useCallback(() => {
+		const {prevToIndex, selectedItem} = mutableRef.current;
+		if (selectedItem) {
+			const selectedItemRect = selectedItem && selectedItem.getBoundingClientRect();
+			mutableRef.current.nextSpotlightRect = {x: selectedItemRect.right, y: selectedItemRect.y};
```

Comment:
Is the `.y` value same as `.top`? then, how about using `top` instead of `y` for consistency.

Reply:
done